frontend/packages/ceph-storage-plugin/src/features.ts

```diff
@@ -139,6 +142,18 @@ export const detectOCS: FeatureDetector = async (dispatch) => {
   }
 };
 
+export const detectManagedODF: FeatureDetector = async (dispatch) => {
+  try {
+    const ns = await k8sGet(NamespaceModel, CEPH_STORAGE_NAMESPACE);
+    if (ns) {
+      const isManagedCluster = ns?.metadata?.labels?.[ODF_MANAGED_LABEL];
+      dispatch(setFlag(ODF_MANAGED_FLAG, !!isManagedCluster));
+    }
+  } catch (error) {
+    dispatch(setFlag(ODF_MANAGED_FLAG, false));
+  }
```

Comment:
```suggestion
  } catch (error) {
    dispatch(setFlag(ODF_MANAGED_FLAG, false));
  }
```
I am not sure if all error means it's not odf_managed. Perhaps we want to see the error status code?

Reply:
I think this would be enough. As this gets triggered only after OCS Operator is installed, which should guarantee the existence of the namespace.